pandas/tests/series/indexing/test_setitem.py

```diff
@@ -191,11 +191,14 @@ def test_setitem_series_object_dtype(self, indexer, ser_index):
         expected = Series([Series([42], index=[ser_index]), 0], dtype="object")
         tm.assert_series_equal(ser, expected)
 
-    @pytest.mark.parametrize("index, exp_value", [(0, 42), (1, np.nan)])
-    def test_setitem_series(self, index, exp_value):
+    @pytest.mark.parametrize(
+        "index, exp_value, warn", [(0, 42, None), (1, np.nan, FutureWarning)]
+    )
+    def test_setitem_series(self, index, exp_value, warn):
         # GH#38303
         ser = Series([0, 0])
-        ser.loc[0] = Series([42], index=[index])
```

Comment:
unrelatedly, this looks sketchy since the RHS is a sequence and the LHS is a scalar